assets/apps/customizer-controls/src/ordering/OrderingComponent.js

```diff
@@ -56,6 +56,26 @@ const OrderingComponent = ({ control }) => {
 						setVisible(true);
 					}
 				});
+
+			if (control.id === 'neve_layout_product_elements_order') {
+				wp.customize('neve_product_content_alignment', (setting) => {
+					setting.bind((val) => {
+						const controlVal = maybeParseJson(
+							control.setting.get()
+						);
+						components.title = __('Title', 'neve');
+						components.price = __('Product Price', 'neve');
+						if (val === 'inline') {
+							components.title =
+								__('Title', 'neve') +
+								' + ' +
+								__('Product Price', 'neve');
+							delete components.price;
+						}
+						updateValue(normalizeValue(controlVal));
+					});
+				});
+			}
```

Comment:
I wonder can we handle these on Neve Pro side, there is an already a logic related that on there, [ref](https://github.com/Codeinwp/neve-pro-addon/blob/master/includes/modules/woocommerce_booster/customizer/shop_page.php#L58-L70)

Reply:
I think, the logic where in the Neve Pro side already handles that process, if I'm correct we can try to fix it. Does it makes sense to you @cristian-ungureanu ?

Reply:
@mskapusuz I tried to fix it from there https://github.com/Codeinwp/neve-pro-addon/issues/2371#issuecomment-1446066514, but I couldn't. The control needs to rerender the items, and I could not trigger that from Pro. I saw we have a similar case for other items like "neve_layout_single_(.+)_elements_order". That's why I added it inside the control.

Reply:
I understood, thank you for the clarifications 🚀